ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/SocialShare/SocialShare.html

```diff
@@ -1,4 +1,6 @@
-<a id="swfMailTo" class="a-link-normal email shareText" title="Share via e-mail" href="">${properties.sharingLabel}</a>
+<div>
```

Comment:
@uditnaagar777 : pl use lowercase in the component name and html. component and file names should be "socialshare" . all in lowercase to be inline to all other components.